pkg/manifest/os.go

```diff
@@ -806,6 +806,13 @@ func usersFirstBootOptions(users []users.User) *osbuild.FirstBootStageOptions {
 			cmds = append(cmds, fmt.Sprintf("mkdir -p %s", sshdir))
 			cmds = append(cmds, fmt.Sprintf("sh -c 'echo %q >> %q'", *user.Key, filepath.Join(sshdir, "authorized_keys")))
 			cmds = append(cmds, fmt.Sprintf("chown %s:%s -Rc %s", user.Name, user.Name, sshdir))
+			if user.Name != "root" {
+				sudoersPath := filepath.Join("/etc", "sudoers.d")
+				sudoersUserFilePath := filepath.Join(sudoersPath, user.Name)
+				cmds = append(cmds, fmt.Sprintf(`sh -c 'echo -e "%s\tALL=(ALL)\tNOPASSWD: ALL" >> %q'`, user.Name, sudoersUserFilePath))
+				cmds = append(cmds, fmt.Sprintf("chmod 440 %s", sudoersUserFilePath))
+				cmds = append(cmds, fmt.Sprintf("restorecon -rvF %s", sudoersPath))
+			}
```

Comment:
Since we might not want this for all ostree-based images (for example, Fedora IoT), we should probably put this particular functionality behind a config option on the image type and pass it down through the OS pipeline and into the `usersFirstBootOptions()` function.
That way we can enable it for RHEL for Edge and not enable it for IoT.  We could also (in the future) make it a user-facing option.

Reply:
done